napari/utils/events/event.py

```diff
@@ -690,6 +702,12 @@ def __getitem__(self, name):
         Note that emitters may also be retrieved as an attribute of the
         EmitterGroup.
         """
+        if name in self._deprecated:
+            print(
+                f"emitter {name} is deprecated, {self._deprecated[name]} provided instead",
+                file=sys.stderr,
+            )
```

Comment:
These should use `warnings.warn(message, category=FutureWarning)`, rather than print.